adam-core/src/main/scala/org/bdgenomics/adam/rdd/GenomicRDD.scala

```diff
@@ -412,6 +420,38 @@ trait GenomicRDD[T, U <: GenomicRDD[T, U]] {
   }
 
   /**
+   * Performs a broadcast inner join between this RDD and data that has been broadcast.
+   *
+   * In a broadcast join, the left RDD is collected to the driver,
```

Comment:
This RDD is the right RDD, correct?
There is no left RDD, rather `broadcastTree` which is an `IntervalArray[ReferenceRegion, X]`.